deps.edn

```diff
@@ -5,5 +5,14 @@
                  :extra-deps {com.github.flow-storm/clojure {:mvn/version "1.11.1-15"}}
                  :jvm-opts ["-Dclojure.storm.instrumentEnable=true"
                             "-Dclojure.storm.instrumentOnlyPrefixes=dev-tester"]}
+           :test {:classpath-overrides {org.clojure/clojure nil}
+                  :extra-deps          {com.github.flow-storm/clojure        {:mvn/version "1.11.1-15"}
+                                        com.github.flow-storm/flow-storm-dbg {:mvn/version "3.17.2"}
```

Comment:
I wouldn't add FlowStorm in Clofidence deps since both will compete for ClojureStorm. Both registers hooks in ClojureStorm and since only one callback for each hook is supported they can't both be used at the same time.

Reply:
Thinking a little deeper while it is true that you can't fully run both at the same time you can still use FlowStorm to debug most of the functionality of Clofidence if you don't call setup-storm and provide the data by hand.